app/assets/javascripts/dashboard.js

```diff
@@ -19,109 +19,56 @@
     return { width : e[ a+'Width' ] , height : e[ a+'Height' ] };
   }
 
-  var setTypeVisibility = function(type, checked) {
-
-    if (type == "all") {
-
-      Object.keys(types).forEach(function(type, i) {
-
-        $('.node-type').prop('checked', checked);
-        setTypeVisibility(type, checked);
-
-      });
-
-    } else {
-
-
-      // record status
-      types[type] = (checked == true);
-
-
-      // record status in browser localStorage
-      if (localStorage) {
- 
-        localStorage.setItem('pl-dash-' + type, checked);
-        // match displayed state to localStorage saved state:
-        $('.node-type-' + type).prop('checked', checked);
- 
-      }
-
-
-      // if all checked?
-      var checked_array = $(".node-type").map(function(i, el) { return $(el).prop('checked'); });
-
-      // if contains some falses:
-      if (checked_array.toArray().indexOf(false) != -1) {
-
-        // if also contains some trues:
-        if (checked_array.toArray().indexOf(true) != -1) {
-
-          $('.node-type-all').prop('indeterminate', true);
-
-        } else {
-
-          $('.node-type-all').prop('checked', false);
-          $('.node-type-all').prop('indeterminate', false);
+  // Adding a wiki type if on mobile
+  if(!viewport().width > 992) {
+    types.wiki = true;
+  }
 
+  $("#notes").click(function(e) {
+    if(!$.trim($(this).find("#notes_container").html()).length) {
```

Comment:
Hmm, what is this section doing? If it's empty, then clicking on it will check the "all" box?

Reply:
Yes, that's what it's doing.